### PR TITLE
IDI-144 added FB & Twitter links, also border to focus state

### PR DESCRIPTION
**Pull request by dayotte**
@acheetham do you have some time to take a quick look at this? thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,7 +69,7 @@
 
                 <div class="outside-in-headerTitle">
                         <span class="outside-in-headerTitle-icon"></span>
-                        <h1>OUTSIDE·IN</h1>
+                        <h1>OUTSIDEIN</h1>
```
Comment (acheetham) [inline index.html]: Was the removal of the 'dash' between 'outside' and 'in' intentional, @dayotte? The dash is still present everywhere else...

File: index.html [hunk 2]
```diff
@@ -194,7 +200,7 @@
 
                         <h4>Who is eligible?</h4>
 
-                        <p>This training program is open to you if you are between the ages of 15-29 and self-identify as having an invisible disability (for example, a learning disability or any other condition that may not be apparent to others) or an episodic disability (for example, MS, lupus, or any other condition where you have good days and bad days). We do not require a doctor’s note or other documents. No prior experience is necessary. However, basic literacy and familiarity with windows operating systems and computers would be an asset.</p>
+                        <p>This training program is open to you if you are between the ages of 15-29 and self-identify as having an invisible disability (for example, a learning disability or any other condition that may not be apparent to others) or an episodic disability (for example, MS, lupus, or any other condition where you have good days and bad days). We do not require a doctors note or other documents. No prior experience is necessary. However, basic literacy and familiarity with windows operating systems and computers would be an asset.</p>
```
Comment (acheetham) [inline index.html]: This apostrophe should remain, I think, if the interpretation is "a note from a doctor"

File: index.html [hunk 3]
```diff
@@ -227,7 +233,7 @@
...
                         <p>The Inclusive Design Research Centre (IDRC) at OCAD University is dedicated to developing policies, systems and practices that support human diversity. The IDRC has successfully led over 50 multi-sector, multi-partner projects dedicated to digital and economic inclusion. The IDRC leads the Inclusive Design Institute (IDI),
-Canada’s first research hub focused on digital inclusion, with 8 post-secondary institutions and over 60 researchers. The IDRC is active in international decision-making bodies ensuring that emerging information and communication systems and practices are inclusively designed from the start and leverage the affordances of digital systems to enable participation in the digital economy.</p>
+Canadas first research hub focused on digital inclusion, with 8 post-secondary institutions and over 60 researchers. The IDRC is active in international decision-making bodies ensuring that emerging information and communication systems and practices are inclusively designed from the start and leverage the affordances of digital systems to enable participation in the digital economy.</p>
```
Comment (acheetham) [inline index.html]: The apostrophe should remain here.

File: index.html [hunk 4]
```diff
@@ -238,7 +244,7 @@
...
                         <p><a href="http://www.riseassetdevelopment.com/" title="RISE Asset Development" target="_blank"> RISE Asset Development </a> </p>
-                        <p>RISE works to empower business owners with access to financing and business supports. We recognize the interdependency of financial well-being to one’s overall quality of life. Rise is committed to improving the lives of people who are unable to secure employment due to mental health or addiction challenges.</p>
+                        <p>RISE works to empower business owners with access to financing and business supports. We recognize the interdependency of financial well-being to ones overall quality of life. Rise is committed to improving the lives of people who are unable to secure employment due to mental health or addiction challenges.</p>
```
Comment (acheetham) [inline index.html]: The apostrophe should remain, here.

**Comment by dayotte** [comment]
@acheetham that should do it. for some reason the dot-hyphen character (alt-8) suddenly stopped showing up in all of my browsers - do you have the same problem?

File: index.html [hunk 5]
```diff
@@ -69,7 +69,7 @@
 
                 <div class="outside-in-headerTitle">
                         <span class="outside-in-headerTitle-icon"></span>
-                        <h1>OUTSIDE·IN</h1>
+                        <h1>OUTSIDE•IN</h1>
```
Comment (acheetham) [inline index.html]: @dayotte, this bullet point (and the one at the bottom) isn't rendering in the browsers – it just looks like an empty space. You'll have to try to revert to the character that was there before. Try copying the old character from the 'diff' view in github, that seems to work.

Comment (simonbates) [inline index.html]: Another option here, would be to use a named HTML entity `&bull;`

See http://www.w3.org/wiki/Common_HTML_entities_used_for_typography

**Comment by dayotte** [comment]
thanks @acheetham, @simonbates, any idea why this would suddenly start happening?

**Pull request merged**